src/components/TodoList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+type Props = {
+  todos: Todo[] | null;
```

Comment:
```suggestion
  todos: Todo[];
```
looks like todos never can be null